lib/cli.ex

```diff
@@ -21,25 +21,34 @@ defmodule Onigumo.CLI do
             {working_dir, switches} = Keyword.pop(switches, :working_dir, File.cwd!())
 
             case switches do
-              [] -> module.main(working_dir)
-              _ -> usage_message()
+              [] ->
+                module.main(working_dir)
+
+              _ ->
+                OptionParser.to_argv(switches)
+                |> Enum.join(", ")
+                |> then(&"incompatible OPTIONS #{&1}")
```

Comment:
```suggestion
                |> Enum.at(0)
                |> then(&"incompatible OPTION #{&1}")
```

This would allow to use the current invalid switch test. That is, without support for multiple invalid switches.

Reply:
Created #279 for a test update that would save us from this.

Reply:
Resolved by merging #279.